utils/connect.js

```diff
@@ -0,0 +1,60 @@
+const { Sequelize, DataTypes, Model } = require('sequelize');
+
+const sequelize = new Sequelize({
+    dialect: 'sqlite',
+    storage: './database.sqlite'
+});
+
+const testConnection = async function() {
```

Comment:
thought: perhaps this should process.exit() so the app doesn't run without a connection?